pipeline_config/pipeline-conditional-processing.yaml

```diff
@@ -0,0 +1,110 @@
+spec: #defined at publish pipeline
+# page classification router 
+# python -m argus.pipeline \
+#   -t argus.processors.ocr_processors.Intake root_docs_path=test_imgs follow_symlinks=true \
+#   -t argus.processors.ocr_processors.PdfTextExtract \
+#   -t argus.processors.ocr_processors.NormalizeImages resample_to_dpi=300 normalize_image_format=.png \
+#   -t argus.processors.ocr_processors.GenericOcr ocr_style=GPUDocTROcr \
+#   -t argus.processors.train_eval_processors.TrainEvalProcessor base_model_name_or_path=/home/mzhao/Data/work/DocAI/test_pipeline/page_classification/page_classifier predict_only=true \
+#   -t argus.processors.train_eval_processors.TrainEvalProcessor base_model_name_or_path=/home/mzhao/Data/work/DocAI/test_pipeline/id_card/model predict_only=true when:EOS "
+# p=pages[0]
+# if p.attributes.get('page_class','w2')=='w9':
+#     action=PROCESS
+# else:
+#     action=SKIP_PAGE
+# " EOS \
+#   -t argus.processors.train_eval_processors.TrainEvalProcessor base_model_name_or_path=/home/mzhao/Data/work/DocAI/test_pipeline/id_card/model predict_only=true when:EOS "
+# p=pages[0]
+# if p.attributes.get('page_class','w9')=='w2':
+#     action=PROCESS
+# else:
+#     action=SKIP_PAGE
+# " EOS \
+#   -t argus.processors.post_processors.generic_post_processor.PostProcessor \
+#   --write_all_steps \
+#   -o page_router_fix
+
+
+  pipeline:
+    steps:
+    - tasks:
```

Comment:
It would be helpful to describe what tasks can be done, and what key-values are available for the user. I would suggest removing generic comments from each individual line to make it look cleaner and easy to follow.

Reply:
I will add some into the yaml file, but most of the above questions I think should go to argus' documentation?